### ISSUE TITLE
credential error during `dl.create_sibling_github`?

### ISSUE THREAD

**Issue by feilong** [templateflow/python-manager#7]
My uploading process crashed during this `dl.create_sibling_github` command:
https://github.com/templateflow/python-manager/blob/master/tfmanager/cli.py#L260-L267

For some reason the traditional `github_login` option keeps causing credential error, and I need to change it to `credential` and use a token to fix it.  I think it might be because the traditional ssh option is obsolete for github and/or datalad?

**Comment by oesteban** [comment]
@feilong you are right. We need to revise this, and we probably want to drop the ssh option.

I hit these issues when addressing #10

**Comment by feilong** [comment]
Makes sense to me.
